encoding/msgpack/msgpack.go

```diff
@@ -0,0 +1,19 @@
+package msgpack
+
+import "github.com/vmihailenco/msgpack/v4"
+
+// Codec encodes/decodes Go values to/from Msgpack.
+// You can use MsgPack instead of creating an instance of this struct.
+type Codec struct{}
+
+// Marshal encodes a Go value to Msgpack.
+func (c Codec) Marshal(v interface{}) ([]byte, error) {
+	return msgpack.Marshal(v)
+}
+
+// Unmarshal decodes a Msgpack value into a Go value.
+func (c Codec) Unmarshal(data []byte, v interface{}) error {
+	return msgpack.Unmarshal(data, v)
+}
+
+var MsgPack = Codec{}
```

Comment:
Ah, here is what's similar to the `encoding.JSON` convenience variable, that's nice. Although this way it might look strange when used: `msgpack.MsgPack`. Do you think this repetition is ok, or could it be better to do it the other way around - use `MsgPack` as actual struct with the methods, and `Codec` as convenience var? I'm currently undecided myself.

Reply:
So, I changed `msgpack.MsgPack` to `msgpack.Codec`.